libs/libarchfpga/src/physical_types.h

```diff
@@ -576,6 +576,23 @@ constexpr int DEFAULT_SWITCH = -2;
  * num_receivers: Total number of input receivers supplied
  * index: Keep track of type in array for easy access
  * logical_tile_index: index of the corresponding logical block type
+ *
+ * In general, the physical tile is a placeable physical resource on the FPGA device,
+ * and it is allowed to contain an heterogeneous set of logical blocks (pb_types).
+ *
+ * Each physical tile must specify at least one sub tile, that is a physical location
+ * on the z axis. This means that a physical tile occupies an (x, y) location on the grid,
```

Comment:
Lets not use z for the third axis (we may have 3D FPGAs in the future) and instead use terminology the tutorial uses (index in the sub_block stack).

Reply:
Done